From af8164d5634911d8684e193291690c23bc4e123b Mon Sep 17 00:00:00 2001
From: Matt Madison <matt@madison.systems>
Date: Sun, 23 Oct 2022 04:41:29 -0700
Subject: [PATCH] core: arm.mk: add -fcommon to cflags

NVIDIA's patches to the core memory management code
introduces some global variables that require this
flag, which is on by default in older versions.

Signed-off-by: Matt Madison <matt@madison.systems>

Upstream-Status: Pending
---
 core/arch/arm/arm.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/core/arch/arm/arm.mk b/core/arch/arm/arm.mk
index dff305b..5e7895f 100644
--- a/core/arch/arm/arm.mk
+++ b/core/arch/arm/arm.mk
@@ -139,7 +139,7 @@ endif
 arm64-platform-cppflags += -DARM64=1 -D__LP64__=1
 arm32-platform-cppflags += -DARM32=1 -D__ILP32__=1
 
-platform-cflags-generic ?= -ffunction-sections -fdata-sections -pipe
+platform-cflags-generic ?= -ffunction-sections -fdata-sections -fcommon -pipe
 platform-aflags-generic ?= -pipe
 
 arm32-platform-aflags += -marm
